src/pages/SetupPlot.vue

```diff
@@ -230,6 +230,7 @@ export default defineComponent({
     await this.updateDriveStats();
     const path = (await tauri.path.dataDir()) + util.appName + util.PLOT_FOLDER;
     this.store.setPlotPath(path);
+    this.createDefaultPlotDir();
```

Comment:
`await`? 
other than, this looks great to me, I don't have any more suggestions :)